src/erlcloud_autoscaling.erl

```diff
@@ -54,12 +54,12 @@ describe_autoscaling_groups(GroupNames) ->
 describe_autoscaling_groups(GroupNames, Config) 
   when is_list(GroupNames), is_record(Config, aws_config) -> 
     %case autoscaling_query(Config, "DescribeAutoScalingGroups", []) of % erlcloud_aws:param_list(GroupNames, "AutoScalingGroupNames.member")) of
-    case autoscaling_query(Config, "DescribeAutoScalingGroups", erlcloud_aws:param_list(GroupNames, "AutoScalingGroupNames.member")) of
-        {ok, Doc} ->
-            Groups = xmerl_xpath:string("/DescribeAutoScalingGroupsResponse/DescribeAutoScalingGroupsResult/AutoScalingGroups/member", Doc),
+    case erlcloud_util:query_all_token(fun autoscaling_query/3, Config, "DescribeAutoScalingGroups", erlcloud_aws:param_list(GroupNames, "AutoScalingGroupNames.member")) of
```

Comment:
i not sure that change of behaviour from "gimme somethings to gimme all the things" is expected here. I encourage you to separate it out.